src/Adapter/Product/Combination/Repository/CombinationRepository.php

```diff
@@ -366,4 +371,17 @@ public function getCombinationIdsByAttributes(ProductId $productId, array $attri
             return new CombinationId((int) $combination['id_product_attribute']);
         }, $result);
     }
+
+    public function updateCombinationStock(ProductId $productId, OutOfStockType $outOfStockType, ShopConstraint $shopConstraint = null): void
+    {
+        $qb = $this->connection->createQueryBuilder();
+        $qb
+            ->update(sprintf('%sstock_available', $this->dbPrefix), 'ps')
```

Comment:
```suggestion
            ->update(sprintf('%sstock_available', $this->dbPrefix), $this->dbPrefix)
```
Ahh, didn't notice before :( sorry this is critical change, needs to be done :sweat: